app/src/main/res/layout/activity_register.xml

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="#FDE54E">
+
+    <ImageView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:src="@drawable/deuvox_logo_black"
+        android:layout_marginTop="25dp"
+        android:layout_marginStart="24dp"/>
+
+    <ImageView
+        android:layout_width="411dp"
+        android:layout_height="520dp"
+        android:layout_alignParentStart="true"
+        android:layout_alignParentTop="true"
+        android:layout_marginTop="256dp"
+        android:background="@drawable/curved_header" />
+
+    <ImageView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="39dp"
+        android:layout_marginTop="18dp"
+        android:src="@drawable/deuvox_amico" />
+
+    <ImageView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="278dp"
+        android:layout_marginTop="98dp"
+        android:src="@drawable/deuvox_ellipse" />
```

Comment:
why is this a different image? shouldn't it just be 1 image?

Reply:
It's two separate image in figma

Reply:
noted 👍